autorest/codegen/templates/operation.py.jinja2

```diff
@@ -27,7 +27,11 @@
 {% endif %}
 {% if operation.parameters.constant|selectattr("implementation", "equalto", "Method")|selectattr("original_parameter", "equalto", None)|selectattr("in_method_code") %}
     {% for constant_parameter in operation.parameters.constant|selectattr("implementation", "equalto", "Method")|selectattr("original_parameter", "equalto", None)|selectattr("in_method_code") %}
+    {% if constant_parameter.serialized_name == "content_type" %}
```

Comment:
I would have imagined:
```
{% if constant_parameter.is_kwarg %}
{{ constant_parameter.serialized_name }} = kwargs.pop("{{ constant_parameter.serialized_name }}", {{ constant_parameter.schema.constant_value }})
```

To be a little more generic, but I understand we have only one kwarg today. Just to be keep in mind if we have more, don't make this place an army of if/if/if/if/if/if/